### PR TITLE
Livestream #121

### PR DESCRIPTION
**Pull request by itsTurner**
This is what I could get done in an attempt to get live stream #121 on the website:

* Update Markdown file name
* Update live stream title
* Omit time from date as per contribution guide
* Added video_id
* Most other things mentioned in contribution guide

### CODE REVIEW

File: _Streams/121-map-reduce-in-javascript.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ - title: "Matter.js"
+   url: "http://brm.io/matter-js/"
+
+videos:
+  - title: "Foundations of Programming in JavaScript"
+    url: "https://www.youtube.com/playlist?list=PLRqwX-V7Uu6Zy51Q-x9tMWIv9cueOFTFA"
+  - title: "CircleCI: Unit Testing and CI"
+    url: "https://www.youtube.com/watch?v=CB7vnoXI0pE"
+  - title: "ES6 Arrow Function"
+    url: "https://www.youtube.com/watch?v=mrYMzpbFz18"
```
Comment (nilsweb) [inline _Streams/121-map-reduce-in-javascript.md]: This video is already available on the website. Please use an internal link instead.

File: _Streams/121-map-reduce-in-javascript.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - title: "Closure in JavaScript"
+    url: "https://www.youtube.com/watch?v=-jysK0nlz7A"
+  - title: "Prototypes in JavaScript"
+    url: "https://www.youtube.com/watch?v=hS_WqkyUah8"
+---
+
+In this live stream, I cover higher order functions in JavaScript and discuss them in the context of arrays: map() and reduce()
```
Comment (nilsweb) [inline _Streams/121-map-reduce-in-javascript.md]: You may want to use proper markdown formatting for the function names (like it was in `_Streams/121-upcoming.md`)

**Comment by itsTurner** [comment]
Thank you for your notes. I have added a new commit with your edits added. Is this what you were looking for?

File: _Streams/121-map-reduce-in-javascript.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ videos:
   - title: "CircleCI: Unit Testing and CI"
     url: "https://www.youtube.com/watch?v=CB7vnoXI0pE"
   - title: "ES6 Arrow Function"
-    url: "https://www.youtube.com/watch?v=mrYMzpbFz18"
+    url: "http://thecodingtrain.com/Tutorials/16-javascript-es6/16.3-arrow-function.html"
```
Comment (nilsweb) [inline _Streams/121-map-reduce-in-javascript.md]: You almost got it. Please just remove the `http://thecodingtrain.com` and `.html` part. The URL would then be: `/Tutorials/16-javascript-es6/16.3-arrow-function`

**Comment by itsTurner** [comment]
Very sorry about that. Here is the complete version.

**Pull request merged**